src/main/resources/xml/schema/facturae/v3_1/Transformer3.1To3.2.xsl

```diff
@@ -0,0 +1,1563 @@
+                    <xsl:if test="boolean(self::AdministrativeCentre/SecondSurname)">
+                        <SecondSurname>
+                            <xsl:value-of select="string(SecondSurname)" />
+                        </SecondSurname>
+                    </xsl:if>
+                    <xsl:if test="boolean(self::AdministrativeCentre/AddressInSpain)">
+                        <AddressInSpain>
+                            <xsl:call-template name="tmpl5_AddressType">
+                                <xsl:with-param name="input" select="AddressInSpain" />
+                            </xsl:call-template>
+                        </AddressInSpain>
+                    </xsl:if>
+                    <xsl:if test="boolean(self::AdministrativeCentre/OverseasAddress)">
+                        <OverseasAddress>
+                            <xsl:call-template name="tmpl6_OverseasAddressType">
+                                <xsl:with-param name="input" select="OverseasAddress" />
+                            </xsl:call-template>
+                        </OverseasAddress>
+                    </xsl:if>
+                    <xsl:if test="boolean(self::AdministrativeCentre/ContactDetails)">
+                        <ContactDetails>
+                            <xsl:call-template name="tmpl7_ContactDetailsType">
+                                <xsl:with-param name="input" select="ContactDetails" />
+                            </xsl:call-template>
+                        </ContactDetails>
+                    </xsl:if>
+                    <xsl:if test="boolean(self::AdministrativeCentre/PhysicalGLN)">
+                        <PhysicalGLN>
+                            <xsl:value-of select="string(PhysicalGLN)" />
+                        </PhysicalGLN>
+                    </xsl:if>
+                    <xsl:if test="boolean(self::AdministrativeCentre/LogicalOperationalPoint)">
+                        <LogicalOperationalPoint>
+                            <xsl:value-of select="string(LogicalOperationalPoint)" />
+                        </LogicalOperationalPoint>
+                    </xsl:if>
+                    <xsl:if test="boolean(self::AdministrativeCentre/CentreDescription)">
+                        <CentreDescription>
+                            <xsl:value-of select="string(CentreDescription)" />
+                        </CentreDescription>
+                    </xsl:if>
+                </AdministrativeCentre>
+            </xsl:if>
+        </xsl:for-each>
+    </xsl:template>
+
+    <xsl:output method="xml" encoding="UTF-8" indent="yes"/>
+    <xsl:template match="/">
+        <fe:Facturae xmlns:fe="http://www.facturae.es/Facturae/2014/v3.2.1/Facturae" xmlns:ds="http://www.w3.org/2000/09/xmldsig#">
+            <xsl:attribute name="xsi:schemaLocation" namespace="http://www.w3.org/2001/XMLSchema-instance">
+                <xsl:value-of select="'http://www.facturae.es/Facturae/2014/v3.2.1/Facturae'"/>
```

Comment:
The scheme must be the 3.2 one

Reply:
It has been a naming error, the transformers modify their respective versions to 3.2.1. I proceed to change the name in the transformers so that you now understand it.

Reply:
I'm afraid the easiest way is to perform small version changes. I'm fixing this.